src/grpc_requests/client.py

```diff
@@ -15,8 +15,23 @@
 
 if sys.version_info >= (3, 8):
     from typing import TypedDict  # pylint: disable=no-name-in-module
+    import importlib.metadata
+
+    def get_metadata(package_name: str):
+        return importlib.metadata.version(package_name)
 else:
     from typing_extensions import TypedDict
+    import pkg_resources
+
+    def get_metadata(package_name: str):
+        return pkg_resources.get_distribution(package_name).version
+
+# Import GetMessageClass if protobuf version supports it
+protobuf_minor_version = get_metadata('protobuf').split('.')[1]
+get_message_class_supported = int(protobuf_minor_version) >= 22
```

Comment:
You might consider checking major version is 4 as well here.

Reply:
Thanks for the catch - I think I had a brain fart thinking that 3 was keeping on a few methods on the minor line as well.